devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -63,11 +63,13 @@ metadata {
 def installed() {
 	createChildDevices()
         refresh()
+	updateDataValue("onOff", "catchall")
```

Comment:
This needs to be before the refresh or else the refresh commands will not be sent to the hub. Also please fix the indentation in these two functions.